nsv13/code/modules/overmap/knpc.dm

```diff
@@ -154,11 +161,11 @@ GLOBAL_LIST_EMPTY(knpcs)
 				continue
 			if(!blocking_firelock.density || blocking_firelock.operating)
 				continue
-			if(blocking_firelock.welded)
+			if((blocking_firelock.welded))
 				break	//If at least one firedoor in our way is welded shut, welp!
 			blocking_firelock.open()	//Open one firelock per tile per try.
 			break
-		for(var/obj/machinery/door/firedoor/blocking_firelock in this_turf)
+		for(var/obj/machinery/door/firedoor/blocking_firelock as() in this_turf)
```

Comment:
Unsafe call, this_turf's contents is not exclusively firelocks.

Reply:
Oh idk why that's there, I guess I got distracted and forgot to remove it when undoing some refactors :p